flink-formats/flink-parquet/src/test/java/org/apache/flink/formats/parquet/ParquetFileSystemITCase.java

```diff
@@ -44,21 +47,18 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** ITCase for {@link ParquetFileFormatFactory}. */
-@RunWith(Parameterized.class)
+@ExtendWith(ParameterizedTestExtension.class)
 public class ParquetFileSystemITCase extends BatchFileSystemITCaseBase {
```

Comment:
It seems some tests are missing here
while running tests for `ParquetFileSystemITCase` flink master it runs 34 tests
for this PR only 31

Reply:
Good catch! I missed the `@Test` in subclass. It also seems that if a subclass `override` the `@TestTemplate` method of the base class, it will not be recognized as a test method. 🤔